src/ptvsd/wrapper.py

```diff
@@ -1557,8 +1557,10 @@ def on_source(self, request, args):
                 pydevd_request,
                 is_json=True)
 
-            body = resp_args['body']
-            self.send_response(request, **body)
+            if resp_args.get('success', True):
```

Comment:
Since `pydevd` now appears to send a proper error response, why is this check still needed?

Reply:
It's necessary to properly propagate it - `send_response` always sends a success response, and old code just propagated the body. Which in error case was just blank, so on the IDE side, you'd get an empty file opened instead of an error.